PublicOnboardingDemo/procedures/Registration.oauthUserRegistration.vail

```diff
@@ -0,0 +1,34 @@
+PROCEDURE Registration.oauthUserRegistration(obj) 
+// Mark this procedure as publicly accessible
+WITH ars_public = true, 
+// Set the execution profiles to admin, regardless of the caller's privilege level
+profiles = ["admin", "admin__system"]
+
+// NOTE: This is the minimum possible privilege we can grant here
```

Comment:
should probably add a note that this method only works from an Application Namespace
```suggestion
// Set the execution profiles to admin, regardless of the caller's privilege level
profiles = ["admin", "admin__system"]

////
//// NOTE:  this method only works if run in an Application Namespace (not a Developer namespace) 
////

// NOTE: This is the minimum possible privilege we can grant here
```